drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -1278,9 +1278,8 @@ void RasterizerStorageGLES2::sky_set_texture(RID p_sky, RID p_panorama, int p_ra
 		if (lod == 1) {
 			//bind panorama for smaller lods
 
-			glActiveTexture(GL_TEXTURE0);
-			glBindTexture(GL_TEXTURE_CUBE_MAP, sky->radiance);
-			shaders.cubemap_filter.set_conditional(CubemapFilterShaderGLES2::USE_SOURCE_PANORAMA, false);
+			// USE_DIRECT_WRITE should be false as we want to properly filter roughness at all LODs other than 0,
+			// even if we're sampling from the panorama instead of the cubemap.
```

Comment:
I would remove all 3 lines of comments and replace with a more descriptive comment about the new process

e.g. "Turn off direct write after first LOD is copied over"

Reply:
Resolved in 763fe84.